src/util/tree.js

```diff
@@ -35,59 +34,71 @@ var path = require( 'path' );
  * before processing any of the items at that level of the tree. The `_pre` and
  * `_post` function will be passed the tree object, and the current directory.
  *
- * @param  {Object} tree    The tree object to traverse.
+ * @param  {Object} root    The tree object to traverse.
  * @param  {Object} methods A mapping of tree keys to processing functions.
  * @param  {String} dir     The root directory of this tree object.
```

Comment:
`{String}` should be `{string}` per the [docs](http://usejsdoc.org/tags-type.html#jsdoc-types)